### DISCUSSION TITLE
v0.2.12

### DISCUSSION THREAD

**Discussion by HEIGE-PCloud** [HEIGE-PCloud/DoIt#169] [General]
### New features
- ✨ add PJAX support (#135) 33243ba @HEIGE-PCloud (#135)
- ✨ add [showcase shortcode](https://hugodoit.pages.dev/theme-documentation-extended-shortcodes/#13-showcase) f493819 @HEIGE-PCloud (#76)
- ➕ add [math shortcode](https://hugodoit.pages.dev/theme-documentation-extended-shortcodes/#14-math) cb920c9 @HEIGE-PCloud  (#144)
- ✨ add more social icons cf5331a 77fd6ef @HEIGE-PCloud (#86) (#153)
- ✨ add support for multi-authors taxonomy 73b2fe0 @xqbumu @HEIGE-PCloud (#98)
- ✨ add support for series taxonomy c74187e @HEIGE-PCloud (#123)
- ✨ add support for [Twikoo comment system](https://twikoo.js.org) 8cf5b1e @HEIGE-PCloud (#97)
- ✨ add support for [Vssue comment system](https://vssue.js.org) d9d8bc1 @HEIGE-PCloud (#157)
- 🔍add support for [Fusejs search](https://fusejs.io/) e0f2696 @HEIGE-PCloud (#104)

### Fixes
- 🐛 fix toc's position in wide pages 7995ea9 @HEIGE-PCloud (#77)
- 🐛 fix overflow for KaTeX formulas e35b922 @HEIGE-PCloud (#83)
- 🐛 fix featured image param 11d375a @HEIGE-PCloud (#117)

### Chores
- ⬆️ update all third-party libraries b2b2b87 @HEIGE-PCloud (#168)
- ♿ improve accessibility @HEIGE-PCloud f480c1f @HEIGE-PCloud (#80)

[v0.2.11...v0.2.12](https://github.com/HEIGE-PCloud/DoIt/compare/v0.2.11...v0.2.12)

<hr /><em>This discussion was created from the release <a href='https://github.com/HEIGE-PCloud/DoIt/releases/tag/v0.2.12'>v0.2.12</a>.</em>

**Comment by wu-yue-yu** [comment]
这个主题真的很喜欢，之前loveit虽然不更了，偶然发现了这个主题，果断继续用下去，真的很谢谢你的积极维护！

**Reply by wu-yue-yu** [reply]
只不过个人还有个小小的建议，就是友链那个shortcode，我还是更喜欢我这种风格：
![友链](https://user-images.githubusercontent.com/59756623/128453734-c2a53f7b-a194-436e-89ac-1a4142129987.png)
这种风格还可以用于当做小的信息卡片hhh
![信息卡片](https://user-images.githubusercontent.com/59756623/128453738-752a28f3-3564-4a75-b0c3-866758173baf.png)
以上是个人的一些建议，大佬已经维护的很好了，我很喜欢这个主题，简单优雅！
友链效果参考自之前在issue里提到的一个[链接](https://blog.233so.com/2020/04/friend-link-shortcodes-for-hugo-loveit/)
